src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITupleExpression.vb

```diff
@@ -531,19 +516,13 @@ Class C
 End Class]]>.Value
 
             Dim expectedOperationTree = <![CDATA[
-IConversionExpression (Implicit, TryCast: False, Unchecked) (OperatorMethod: Function C.op_Implicit(x As (System.Int32, System.String)) As C) (OperationKind.ConversionExpression, Type: C) (Syntax: '(0, Nothing)')
-  Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: True) (MethodSymbol: Function C.op_Implicit(x As (System.Int32, System.String)) As C)
-  Operand: 
-    IConversionExpression (Implicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: (System.Int32, System.Object)) (Syntax: '(0, Nothing)')
```

Comment:
>IConversionExpression (Implicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: (System.Int32, System.Object)) (Syntax: '(0, Nothing)') [](start = 4, length = 158)

It feels like there is something wrong with this conversion node. It converts from (System.Int32, System.Object) to (System.Int32, System.Object), but says it is not an identity conversion. We probably shouldn't have this node in the tree at all. Please open an issue to follow up. We will have to create a new unit-test that gets the tree for the entire Dim statement.

Reply:
https://github.com/dotnet/roslyn/issues/22587